src/discussions/posts/post-editor/PostEditor.jsx

```diff
@@ -188,18 +201,6 @@ function PostEditor({
     );
   }
 
-  const initialValues = {
-    postType: post?.type || 'discussion',
-    topic: post?.topicId || topicId || nonCoursewareTopics?.[0]?.id,
-    title: post?.title || '',
-    comment: post?.rawBody || '',
-    follow: isEmpty(post?.following) ? true : post?.following,
-    anonymous: allowAnonymous ? false : undefined,
-    anonymousToPeers: allowAnonymousToPeers ? false : undefined,
-    editReasonCode: post?.lastEdit?.reasonCode || '',
-    cohort: post?.cohort || 'default',
-  };
-
```

Comment:
what happens? when you open the editor to edit posts/comments/responses?

Reply:
It will work as it was working before. I just relocate this constant.